test/asdf_elvish.bats

```diff
@@ -3,12 +3,15 @@
 load test_helpers
 
 setup() {
+  # shellcheck disable=SC1007
   export XDG_CONFIG_HOME= XDG_DATA_HOME= XDG_DATA_DIRS=
 
   local version=
   version=$(elvish -version)
 
+  # shellcheck disable=SC1007
```

Comment:
Change to

```
  local ver_major=
  local ver_minor=
  local ver_patch=
```

for same reasons as above.

Reply:
:+1: